global.yaml

```diff
@@ -0,0 +1,52 @@
+global:
+    earth_color: '#82fd88'
+    water_color: '#2594e4'
+    landuse_color: '#46eba7'
+    text_font_family: 'Montserrat'
+    text_stroke_color: 'white'
+    text_stroke: { color: global.text_stroke_color, width: 2px }
+    text_fill_color: 'black'
+    building_color: '#caffd2'
+    building_outline: black
+    road_color: '#4da39a'
+    road_outline: black
+    path_color: global.road_color
+    path_outline: global.road_outline
```

Comment:
Shouldn't the globals that only apply to the layers style not be moved there?

I understand that global.yaml is designed to be the "shared" code. Then, path_color etc. wouldn't be shared though.

Reply:
This is now deprecated because there is now only one imported file which needs **all** globals